var/spack/repos/builtin/packages/plasma/package.py

```diff
@@ -48,22 +55,33 @@ class Plasma(MakefilePackage):
     conflicts("%xl_r")
 
     patch("remove_absolute_mkl_include.patch", when="@17.1")
+    patch("add_netlib_lapacke_detection.patch", when="@17.1")
 
     def edit(self, spec, prefix):
         # copy "make.inc.mkl-gcc" provided by default into "make.inc"
         open("make.inc", "w").write(open("make.inc.mkl-gcc").read())
 
         make_inc = FileFilter("make.inc")
 
-        if not spec.satisfies("^mkl"):
+        ld_flags = self.spec["lapack"].libs.ld_flags + " " + \
+            self.spec["blas"].libs.ld_flags
+
+        if '~shared' in self.spec:
+            make_inc.filter("-fPIC", "")  # not using fPIC
+            ld_flags += " -lm"
+
+        if "^mkl" not in spec:
             make_inc.filter("-DPLASMA_WITH_MKL", "")  # not using MKL
 
+        if "^netlib-lapack" in spec:
+            ld_flags += " -llapacke -lcblas"
```

Comment:
i think ideally this should go into 
```
 @property
    def lapack_libs(self):
```
 of `netlib-lapack` and be returned when a user asks for additional key, for example`lapacke`.  Look at at `hdf5` package as an example of how to use the extra key.

Then there is one place to manage the lapacke libs for `netlib-lapack` and all packages in Spack can use it.

Reply:
p.s. for `hdf5` one can ask `spec['hdf5:hl,cxx'].libs`, what we'll have here is something like `spec['lapack:c'].libs` to get `c` interface to Lapack and `spec['blas:c'].libs` for BLAS.